src/filter.ts

```diff
@@ -227,7 +228,16 @@ export class Filter {
      * @param {boolean} enabled
      */
     public setUnreadThreadNotifications(enabled: boolean): void {
-        setProp(this.definition, "room.timeline.unread_thread_notifications", !!enabled);
+        this.definition = {
+            ...this.definition,
+            room: {
+                ...this.definition?.room,
+                timeline: {
+                    ...this.definition?.room?.timeline,
+                    [UNREAD_THREAD_NOTIFICATIONS.name]: !!enabled,
```

Comment:
This'll now always use the unstable. If your HS only has stable support and not unstable it'll cause /sync to explode.

Reply:
I might be misunderstanding the use of `UnstableValue` then 🤔